pkg/imgpkg/cmd/push.go

```diff
@@ -56,11 +59,20 @@ func (po *PushOptions) Run() error {
 		return err
 	}
 
+	err = po.validateFlags()
+	if err != nil {
+		return err
+	}
+
 	var imageURL string
 
 	isBundle := po.BundleFlags.Bundle != ""
 	isImage := po.ImageFlags.Image != ""
 
+	if po.LabelFlags.Labels == nil {
```

Comment:
Do we need this check here if we set a default in the LabelsFlags?

Reply:
I didn't think so at first, but there were some nil map assignment errors in some of the tests when I didn't have this. Having this here also prevented having to do the check for both Images and Bundles separately since they seem to both start from this code path.

Reply:
Removing this from here in favor of having it in the bundle `Push()` method as per the[ above thread]([pkg/imgpkg/bundle/contents.go](https://github.com/carvel-dev/imgpkg/pull/552/files/a6db6ee2ab2f394a760c1ca5259b8a6500381255#diff-2d42d9f47abf1127955d8688255d683771a9c39d03d08d3f84a44c43a599c32f))